api/main.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"context"
```

Comment:
There's tabs in this file, is that a Go thing?

Reply:
Yah, `gofmt` changes it to tabs automatically.